src/sql/workbench/contrib/notebook/browser/notebookLinkHandler.ts

```diff
@@ -51,7 +51,7 @@ export class NotebookLinkHandler {
 			this.isAbsolutePath = this._link.attributes['is-absolute']?.nodeValue === 'true' ? true : false;
 			this.isMarkdown = this._link.attributes['is-markdown']?.nodeValue === 'true' ? true : false;
 			this.isEncoded = this._link.attributes['is-encoded']?.nodeValue === 'true' ? true : false;
-			this._isFile = this._link.protocol === 'file:';
+			this._isFile = this._link.protocol === 'file:' || this._link.protocol === 'vscode-file:';
```

Comment:
Use [file](https://github.com/Microsoft/azuredatastudio/blob/main/src/vs/base/common/network.ts#L41) and [vscodeFileResource](https://github.com/Microsoft/azuredatastudio/blob/main/src/vs/base/common/network.ts#L95)